react/features/toolbox/components/AudioRouteButton.js

```diff
@@ -0,0 +1,153 @@
+// @flow
+
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import {
+    findNodeHandle,
+    requireNativeComponent,
+    NativeModules,
+    Platform,
+    View
+} from 'react-native';
+import { connect } from 'react-redux';
+
+import ToolbarButton from './ToolbarButton';
+
+import { openDialog } from '../../base/dialog';
+import { AudioRoutePickerDialog } from '../../mobile/audio-mode';
+
+/**
+ * Define the {@code MPVolumeView} React component. It will only be available
+ * on iOS.
+ */
+const MPVolumeView = requireNativeComponent('MPVolumeView', null);
+
+/**
+ * Style required to hide the {@code MPVolumeView} view, since it's displayed
+ * programmatically.
+ */
+const HIDE_VIEW_STYLE = { display: 'none' };
+
+/**
+ * A toolbar button which triggers an audio route picker when pressed.
+ */
+class AudioRouteButton extends Component<*> {
+    /**
+     * AudioRouteButton component's property types.
+     *
+     * @static
+     */
+    static propTypes = {
```

Comment:
The new flow version finally allows us to have React's `static propTypes` converted to flow's `type Props`. Please do for the new files (including this PR) because we'll be retroactively and iteratively updating the existing source code.